test/python/expected/sql_validation_framework/expected_create.out

```diff
@@ -79,6 +79,8 @@ Could not find tests for table sys.versions
 Could not find tests for type sys.bpchar
 Could not find tests for view babelfish_has_perms_by_name_permissions
 Could not find tests for view sys.all_sql_modules_internal
+Could not find tests for view sys.asymmetric_keys
```

Comment:
Add tests for the views which you have added though it's a dummy view and returns NULL.

Reply:
Added